packages/main/src/themes/CheckBox.css

```diff
@@ -175,6 +175,7 @@ https://github.com/philipwalton/flexbugs/issues/231
 	height: var(--_ui5_checkbox_icon_size);
 	color: currentColor;
 	cursor: default;
+    position: absolute;
```

Comment:
It turns out this misaligns the checkmark icon in IE:

<img width="43" alt="Screenshot 2021-08-18 at 12 30 46" src="https://user-images.githubusercontent.com/15702139/129874536-6ed6f098-74d6-4ad8-9c56-b1df37d76596.png">